### ISSUE TITLE
Spell Check on `Rmd` files.

### ISSUE THREAD

**Issue by bms63** [pharmaverse/admiralci#33]
I've noticed some misspelling ins the vignettes and I think we don't have any spell checking running over them.  Looking at the spell check we are excluding `.Rmd`.  Could we stop excluding these checks or are the html files checked somewhere down the road??

https://github.com/pharmaverse/admiralci/blob/4f0d6f74e9ae96a40fbb6f90b192dbefe6c87d9f/.github/workflows/spellcheck.yml#L94

**Comment by galachad** [comment]
I totally agree, the `spelling` package do not support exclusion and it's hard to read checks without having the full `WORDLIST` available.

Btw. The exclusion inside job is done in the way that is not recommended to reproduce:
```
lapply(Sys.glob(exclusions), file.remove)
```
